### ISSUE TITLE
Examples in non-provided languages should be displayed.

### ISSUE THREAD

**Issue by jamescummings** [TEIC/Stylesheets#273]
As mentioned in: https://github.com/TEIC/TEI/issues/1651

The element reference pages only put out examples that are in the language of the element reference page being generated (e.g. only puts out chinese on the chinese pages).  However, there is a small list of languages that we provide other sets of reference pages for.  The generating code for a reference page should put out:

- any examples from the spec file in that language (e.g. en, de, es, it, fr, ja, ko, zh-TW)
- and any examples from the spec file that are in other languages which are not one of the other languages we provide: en, de, es, it, fr, ja, ko, zh-TW should also be provided

The net effect of this is that examples from the spec files in languages other than these would appear on all language pages. e.g. the `<metamark>` page doesn't have an example because its example is in norwegian. However, you don't need to know norwegian to understand the example, so it should be present on all pages in all languages.

My guess would be that this might be a change in https://github.com/TEIC/Stylesheets/blob/dev/odds/guidelines.xsl 
or in one of the related odds stylesheets.

**Comment by lb42** [comment]
Might be simpler just to provide a non-Norwegian example! (Or pretend that this one isn't)

**Comment by jamescummings** [comment]
I have provided a non-Norwegian example, at https://github.com/TEIC/TEI/commit/a32b392fbe8facc78328c291cce9c2aac459799c, but the question remains how many other exemplum do we have where the example is never being used because it isn't one of the languages we display?

**Comment by lb42** [comment]
Easy enough to produce a list of counts of exempla per element subdivided by language...

**Comment by ebeshero** [comment]
I think our processing of examples has changed since this ticket was opened...can we verify that and see if this ticket may be closed?

**Comment by sydb** [comment]
I do not believe this particular problem has been fixed, and thus I think this ticket should remain open. (As evidence, the Norwegian example is not on the metamark tagdoc page.)
